doc/features.md

```diff
@@ -126,6 +126,9 @@ The following live templates are available:
 - `comp` – computed property
 - `compset` – computed property with setter
 - `compx` – computed property method
+- `test` - qunit test
+- `pt` - pauseTest
+- `then` - andThen
```

Comment:
could you make sure to `–` as the separator too instead of `-`? sorry for being so picky 😉